File: co.edu.uniandes.fixml.toCSharp/test/text2.xml/Sndr.cs
namespace text2.xml;



// Class for tag Sndr
class Sndr {

	string ID;
	string Sub;
	string Loc;
	
	
	// default constructor
	
	Sndr ( ) {   
		this.ID = "String";
		this.Sub = "String";
		this.Loc = "String";
		
		
	}
	
	Sndr ( String ID, String Sub, String Loc ) {
		this.ID = ID;
		this.Sub = Sub;
		this.Loc = Loc;
		
	}
	
	
} 

File: co.edu.uniandes.fixml.toC++/test/text2.xml/Sndr.cpp
#include <iostream> 

using namespace std;

class Sndr {

	private:
  	string ID_;
  	string Sub_;
  	string Loc_;
 	
 	public: 

  	// default constructor
	Sndr () {
		ID_ = "String";
		Sub_ = "String";
		Loc_ = "String";
  	
  	}
  
  	// by parameters
  	Sndr (		string ID_p, string Sub_p, string Loc_p){
	  
		ID_ = ID_p;
		Sub_ = Sub_p;
		Loc_ = Loc_p;
	  
  	}
};